### PR TITLE
Histogram example - Java

### PR DESCRIPTION
**Pull request by dominikl**
Adds a histogram example to the Java training examples.

See also:
- https://trello.com/c/5SjwyaM4/112-histogram-api
- https://github.com/openmicroscopy/openmicroscopy/pull/5565

### CODE REVIEW

**Comment by waxenegger** [comment]
Builds and shows without errors:

image

File: omero/developers/Java.rst [hunk 1]
```diff
@@ -339,6 +339,25 @@ This is useful when you need the pixels intensity.
         store.close();
     }
 
+-  **Retrieve a histogram.**
+
+::
+
+    RawPixelsStorePrx store = gateway.getPixelsStore(ctx);
```
Comment (jburel) [inline omero/developers/Java.rst]: in the example you have the store initialised in the ``try/catch``
you should unify that

Comment (dominikl) [reply omero/developers/Java.rst]: And I just noticed that the getHistogram method is already in the RawDataFacility, which should be used preferably. I have to update the example and doc again.

**Comment by waxenegger** [comment]
HMTL is generated properly and changes are in sync with actual code examples.

**Pull request merged**